docker/web/check_db.py

```diff
@@ -0,0 +1,28 @@
+import socket
+import time
+import argparse
+""" Check if port is open, avoid docker-compose race condition """
+
+parser = argparse.ArgumentParser(description='Check if port is open, avoid\
+                                 docker-compose race condition')
```

Comment:
I will leave this comment for the future: regarding code style decision, will we break long/multiple arguments like this?